.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
please revert this change

Reply:
Thank you. BuildConfiguration: I changed it automatically